assets/js/components/ViewSelector.js

```diff
@@ -235,7 +235,12 @@ module.exports = class ViewSelector {
 
   insertSideBySideListItem() {
     const $list = this.doc.querySelector('.view-selector__list');
-    const $jumpList = this.doc.querySelector('.view-selector__list-item--jump');
+    const $jumpList = this.doc.querySelector('.view-selector__list-item--figures + .view-selector__list-item');
```

Comment:
it's not the jump list anymore but may be the repro link? Is it just the `$attachBefore` argument now?